Description: Fixes calls to iteritems()
 iteritems() is gone away from Python3, fixes that using list(dict.items())
 instead.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-04-29

--- python-proliantutils-2.1.0.orig/proliantutils/hpssa/constants.py
+++ python-proliantutils-2.1.0/proliantutils/hpssa/constants.py
@@ -32,7 +32,7 @@ RAID_10_ADM = '10ADM'
 
 RAID_LEVEL_INPUT_TO_HPSSA_MAPPING = {RAID_50: '50', RAID_60: '60'}
 RAID_LEVEL_HPSSA_TO_INPUT_MAPPING = {
-    v: k for k, v in RAID_LEVEL_INPUT_TO_HPSSA_MAPPING.iteritems()}
+    v: k for k, v in list(RAID_LEVEL_INPUT_TO_HPSSA_MAPPING.items())}
 
 INTERFACE_TYPE_MAP = {'SCSI': INTERFACE_TYPE_SCSI,
                       'SAS': INTERFACE_TYPE_SAS,
--- python-proliantutils-2.1.0.orig/proliantutils/hpssa/objects.py
+++ python-proliantutils-2.1.0/proliantutils/hpssa/objects.py
@@ -231,7 +231,7 @@ class Server(object):
         raid_info = _convert_to_dict(config)
         self.controllers = []
 
-        for key, value in raid_info.iteritems():
+        for key, value in list(raid_info.items()):
             self.controllers.append(Controller(key, value, self))
 
         self.last_updated = time.time()
@@ -313,7 +313,7 @@ class Controller(object):
         self.raid_arrays = []
 
         unassigned_drives = properties.get('unassigned', dict())
-        for key, value in unassigned_drives.iteritems():
+        for key, value in list(unassigned_drives.items()):
             self.unassigned_physical_drives.append(PhysicalDrive(key,
                                                                  value,
                                                                  self))
